client/parser/tinymembench_parser.py

```diff
@@ -24,48 +24,25 @@ def tinyResult(content, outfp):
 	dic['tiny_latency']['Hugepage dual random read'] = []
 	dic['tiny_latency']['No Hugepage single random read'] = []
 	dic['tiny_latency']['No Hugepage dual random read'] = []
-
+	dic_list = ['C copy backwards','C copy','C copy prefetched 32B','C copy prefetched 64B','C 2-pass copy','C 2-pass copy prefetched 32B','C 2-pass copy prefetched 64B','C fill','standard memcpy','standard memset']
 
 	outfp.write('OPERATIONS\t\t\t\t\t:\tBANDWIDTH\n')
```

Comment:
Please don't use "/t" instead use the number of space defined by numbers.
Please add heading section for memory bandwidth. Refer your parser log.